install.php

```diff
@@ -1288,14 +1291,31 @@ function BuildFileList(){
 		Config::UpdateParameter( 'LDAPRackRequest', $_REQUEST['LDAPRackRequest']);
 		Config::UpdateParameter( 'LDAPRackAdmin', $_REQUEST['LDAPRackAdmin']);
 		Config::UpdateParameter( 'LDAPContactAdmin', $_REQUEST['LDAPContactAdmin']);
+		Config::UpdateParameter( 'LDAPBulkOperations', $_REQUEST['LDAPBulkOperations']);
 		Config::UpdateParameter( 'LDAPSiteAdmin', $_REQUEST['LDAPSiteAdmin']);
+
+		if ( AUTHENTICATION == "AD") {
+			Config::UpdateParameter( 'LDAPUserSearch', $_REQUEST['LDAPUserSearch']);
+		}
+
+		// Refresh config.
+		$config->Config();
+
+		$_SESSION['ldapauthset'] = true;
```

Comment:
This is how we determine if the AD/LDAP setup has been completed during installation, allowing the "Complete" tab to be shown.